recipes/beautysh/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "beautysh" %}
+{% set version = "3.10" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: c7058406aaf13d397c1eb54b7d6b8577cd282ad0f97bac985fa314ffe80179b0
+    folder: dist
+  - url: https://github.com/bemeurer/{{ name }}/archive/{{ version }}.tar.gz
+    sha256: 17f452b7206a8c121c7641e9595fd2a7ff5b3779ad4dca476f107a60126a753f
+    folder: src
+
+build:
+  noarch: python
+  number: 0
+  script: cd dist && {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  entry_points:
+    - beautysh = beautysh.beautysh:main
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+
+test:
+  imports:
+    - beautysh
+  commands:
+    - beautysh --help
+
+about:
+  home: https://github.com/bemeurer/beautysh
+  license: MIT
+  license_family: MIT
+  license_file: src/LICENSE
+  summary: A Bash beautifier for the masses.
+
+extra:
+  recipe-maintainers:
+    - bollwyvl
```

Comment:
@conda-forge-linter looks fine to me...

Reply:
@bollwyvl Try `recipe-maintainers: [bollywyvl]`